services/EventAPIService.js

```diff
@@ -0,0 +1,71 @@
+const axios = require("axios");
+
+// A service class for all REST API operations
+class EventAPIService {
+  #axiosInstance;
+  #tokenServiceInstance;
+  constructor(tokenService) {
+    // Set Axios baseURL to 100ms API BaseURI
+    this.#axiosInstance = axios.create({
+      baseURL: "https://api.100ms.live/v2",
```

Comment:
NIT: can we make this url as const, we can modify the constants easily as they might be stored in some common place.

Reply:
+1

Reply:
Thank you for the suggestion. Fixed it!